ss58-registry.json

```diff
@@ -136,6 +136,15 @@
 			"standardAccount": "*25519",
 			"website": "https://www.substratee.com"
 		},
+		{
+			"prefix": 14,
+			"network": "totem",
+			"displayName": "Totem",
+			"symbols": ["XTX"],
+			"decimals": [0],
```

Comment:
`0` decimals are right?

Reply:
Yes this is intentional. 

In the UI the user will display in their own currency, but under the hood all values are stored at a system derived exchange rate to the XTX currency. XTX therefore functions in the precise context of a "unit of account" for accounting purposes. 

It also makes it possible to add cost entires into the expense accounting for transaction fees.